.coveragerc

```diff
@@ -0,0 +1,29 @@
+[html]
+show_contexts = true
+skip_covered = false
+
+[paths]
+source =
+  mysite
```

Comment:
This section is not needed. It's used to map files in your repository with their site-packages copies. But since a Django project isn't `pip install`ed, there should be no need to configure this mapping.

Reply:
Marking this as resolved.